### ISSUE TITLE
NESHawk sound

### ISSUE THREAD

**Issue by DJIncendration1** [TASEmulators/BizHawk#3164]
In NESHawk/SubNESHawk, some of the audio is slightly inaccurate, when compared to console. Examples of this include the pipe sound from Super Mario Bros, and the knock-down and ducking sound from Mike Tyson's Punch-Out!! It is inaccurate when I start the game with the core, no other steps needed to reproduce the issue.

**Comment by Yave-Yu** [comment]
RoboCop 2's message sound is slightly inaccurate to real hardware too.

**Comment by ThunderAxe31** [comment]
Is there any emulator that emulates those audio effects better?

**Comment by DJIncendration1** [comment]
Yes. FCEUX (2.3.0 and later), Mesen and BizHawk's QuickNES core all emulate sound accurately.

**Comment by DJIncendration1** [comment]
Also, the sound issues appear to be on the square 2 channel.

**Comment by Shinobi-87** [comment]
Kind of off topic, but could maybe Nestopia UE get waterboxed? Its super fast and very compatible.

**Comment by DJIncendration1** [comment]
Nestopia UE is also inaccurate in terms of sound for some games, including MTPO.

**Comment by Shinobi-87** [comment]
> Mesen is pretty fast, too. Better yet, it's accurate (aside from the additional frame like FCEUX.)

Mesen is my goto but i just assume they want lighter weight.
